src/main/java/co/inventrosoft/springboottask/repository/TeamRepositoryFileImpl.java

```diff
@@ -0,0 +1,72 @@
+package co.inventrosoft.springboottask.repository;
+
+
+import co.inventrosoft.springboottask.model.Team;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Repository;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+
+@Repository
+public class TeamRepositoryFileImpl implements TeamRepository{
+    private final static String teamsFile = "teams.json";
+    private final ObjectMapper mapper;
+
+    public TeamRepositoryFileImpl(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
+
+    private List<Team> findAll() {
+        List<Team> teams = null;
```

Comment:
It is bad idea to return `null` instead of empty collection.
You have just created new NPE :)